Jenkinsfile

```diff
@@ -6,6 +6,7 @@ pipeline {
             when {
                 anyOf {
                     branch 'developer-tmp'; branch pattern: 'PR-\\d+', comparator: 'REGEXP'
+                    branch 'master'; branch pattern: 'PR-\\d+', comparator: 'REGEXP'
```

Comment:
Not needed. Tests are run on developer-tmp constantly. Once master gets built tests will be run on that aswell. This just slows the build process down

Reply:
ah ok, I just thought that it was missed